src/lib/y2storage/autoinst_issues/issue.rb

```diff
@@ -7,6 +7,9 @@ module AutoinstIssues
     class Issue
       include Yast::I18n
 
+      # @return [Object] section where the problem was detected (see {AutoinstProfile})
```

Comment:
well, to be honest documenting only Object as parent is not much helpful. is there any methods on top of pure Object that it have in common? I think for documentation purpose it would be helpful to have class that define common methods for all sections.

Reply:
currently it looks like at least parent and section_name is must have, not?

Reply:
Well, that's partially true: these classes will only hold a reference to those objects (to be used by others later). But it's true that maybe something like `#parent,#section_name` would be better than `Object`.